### PR TITLE
Feature/image editing

### PR DESCRIPTION
**Pull request by Gouterman**

### CODE REVIEW

File: themes/Rozier/Resources/routing/documents.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+documentsRetouchPage:
+    path:     /retouch/{documentId}
```
Comment (ambroisemaupate) [inline themes/Rozier/Resources/routing/documents.yml]: ajust

File: themes/Rozier/Resources/views/documents/navBar.html.twig [hunk 2]
```diff
@@ -5,18 +5,23 @@
...
+            <a href="{{ path('documentsMetaPage', editRouteParams) }}">{% trans %}document.metas{% endtrans %}</a>
+        </li>
+        <li class="{% if current == 'preview' %}uk-active{% endif %}">
+            <a href="{{ path('documentsPreviewPage', editRouteParams) }}">{% trans %}document.preview{% endtrans %}</a>
+        </li>
+        <li class="{% if current == 'usage' %}uk-active{% endif %}">
+            <a href="{{ path('documentsUsagePage', editRouteParams) }}">{% trans %}document.usage{% endtrans %}</a>
+        </li>
+        {% if document and document.image %}
+            <li class="{% if current == 'retouch' %}uk-active{% endif %}">
```
Comment (ambroisemaupate) [inline themes/Rozier/Resources/views/documents/navBar.html.twig]: ajust

File: themes/Rozier/Resources/views/documents/retouch.html.twig [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{% if not head.ajax %}{% set baseTemplate = '@Rozier/base.html.twig' %}{% else %}{% set baseTemplate = '@Rozier/ajaxBase.html.twig' %}{% endif %}{% extends baseTemplate %}
+
+{% block content %}
+    <section class="content-global manage-document">
+        <header class="content-header header-document header-document-edit">
+            <h1 class="content-title document-edit-title">{{ "edit.document.%name%"|trans({'%name%': document.getFilename})|truncate(25, true, '[…]') }}</h1>
+            {% include '@Rozier/documents/navBar.html.twig' with {'document':document, current:'retouch'} %}
```
Comment (ambroisemaupate) [inline themes/Rozier/Resources/views/documents/retouch.html.twig]: ajust

File: themes/Rozier/Controllers/Documents/DocumentsController.php [hunk 4]
```diff
@@ -148,6 +150,98 @@ public function indexAction(Request $request, $folderId = null)
 
     /**
      * @param Request $request
+     * @param $documentId
+     * @return JsonResponse|Response
+     */
+    public function retouchAction(Request $request, $documentId)
```
Comment (Gouterman) [inline themes/Rozier/Controllers/Documents/DocumentsController.php]: adjust

File: themes/Rozier/Resources/routing/documents.yml (hunk 1 above)
Comment (Gouterman) [inline themes/Rozier/Resources/routing/documents.yml]: adjust

File: themes/Rozier/Resources/views/documents/retouch.html.twig (hunk 3 above)
Comment (Gouterman) [inline themes/Rozier/Resources/views/documents/retouch.html.twig]: adjust

**Pull request merged**